cernopendata/modules/fixtures/data/docs/delphi-guide-docker/delphi-guide-docker.md

```diff
@@ -1,80 +1,72 @@
-In this guide, we will assume to be running on a Linux desktop, which has podman and podman-docker installed. Some of the applications require a graphical interface, namely the CERNLIB applications paw, paw++ and kxterm, as well as the event display.
+In this guide, we will assume to be running on a Linux desktop, which has the `podman` and `podman-docker` packages installed. Some of the applications require a graphical interface,
+namely the CERNLIB applications `paw`, `paw++` and `kxterm`, as well as the [DELPHI event display](delphi-guide-delgra).
 
-If you want to run on any data samples, please download them first. You can then attach them using the -v option of the `podman run` command. Please check the podman documentation for more details.
+If you want to run on any data samples, please download them first. You can then attach them using the -v option of the `podman run` command. Please check the podman documentation for more details. The `Alma9` image we use in this guide can also access the data directly from the `EOS` file system.
 
-The image is configured with a local user called delphi. To start the container on a Linux based system, use:
+The image is configured with a local user called `delphi`. For the commands below, we'll use the following conventions:
+
+* a prompt `$ xyz` indicates that the command `xyz` is to be run on your local machine or VM.
+* a prompt `[delphi ~] $ xyz` indicates that the command `xyz` is to be run inside the started container.
+
+To start the container on a Linux based system, use:
 
 ```
-xhost + local:docker
-docker run --privileged --rm -it -e DISPLAY --network host -v /tmp/.X11-unix:/tmp/.X11-unix -v ~/.Xauthority:/home/delphi/.Xauthority --user delphi gitlab-registry.cern.ch/delphi/deployment/delphi/al9_64 /bin/bash -l
+$ xhost + local:docker
+$ docker run --privileged --rm -it -e DISPLAY --network host -v /tmp/.X11-unix:/tmp/.X11-unix -v ~/.Xauthority:/home/delphi/.Xauthority --user delphi gitlab-registry.cern.ch/delphi/deployment/delphi/al9_64 /bin/bash -l
 ```
 
-This command will download the Alma9 based container and create a login shell for the DELPHI user. This image comes with support for EOS: when launched on a system which supports fuse file systems, the DELPHI data will be available inside the container beneath the path `/eos/opendata/delphi`. Note that due to the --rm option, the container will be destroyed when you exit it.
+This command will download the Alma9 based container and create a login shell for the DELPHI user.
+
+This image comes with support for `EOS`: when launched on a system which supports fuse file systems, the DELPHI data will be available inside the container beneath the path `/eos/opendata/delphi`. Note that due to the --rm option, the container will be destroyed when you exit it, and all the work done inside the container will be lost.
 
 <p><center><img src="/static/docs/delphi-guide-docker/delphi-container-start.png" width="60%"></center></p>
```

Comment:
BTW the image uses `delphi@detritus`  prompt.

Besides this host name being a bit funny :wink: it would be nice if the prompt in the image corresponded to the prompt used in the guide, so you could perhaps retake the picture to remove "detritus"?